### PR TITLE
Migrate metadata mapping

### PR DESCRIPTION
**Pull request by luzip665**
Some metadata mappings are not like the standard.
Sources:
https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html
https://wiki.hydrogenaud.io/index.php?title=Tag_Mapping
https://docs.kde.org/trunk5/en/kid3/kid3/commands.html#frame-list

See also discussion at #682 

For flac and ogg there is new mapping and migration for bpm and comment.

### CODE REVIEW

File: xl/main.py [hunk 1]
```diff
@@ -644,6 +644,11 @@ def __init(self):
 
         engine.migrate()
 
+        # Migrate flac tempo tag
```
Comment (sjohannes) [inline xl/main.py:647]: Is this just for FLAC or also Ogg (/Vorbis) files?

**Review by sjohannes — COMMENTED** [review]
I suppose the problem here is that the migration only works for files in the collection. But I'm not sure I have a better alternative.

File: xl/main.py (hunk 1 above)
Comment (luzip665) [reply xl/main.py:647]: It's for both. 
But the migration only sets the setting to true in case of a first run and false otherwise.

**Comment by luzip665** [comment]
@sjohannes 
The migration works only with files in the collection.
But how could we change files outside of it? I mean, we could offer a context menu entry to migrate each track individually, but I'm not sure if this is necessary.

**Comment by sjohannes** [comment]
I was thinking that we'd notify the user when we see an external file with the 'wrong' tags. But I don't know if it's worth the effort.

**Pull request merged**